docs/machine-learning/tutorials/tensorflow-transfer-learning.md

```diff
@@ -0,0 +1,337 @@
+
+Finally, these image features will be fed to a Logistic regression algorithm which will learn how to classify different sets of image features.
+
+## DataSet
+
+There are two data sources: the `tsv` file and the image files.  The tags.tsv file contains two columns: the first one is defined as `ImagePath` and the second one is the `Label` corresponding to the image. The following example file does not have a header row, and looks like this:
+
+```tsv
+broccoli.jpg	broccoli
+pizza.jpg	pizza
+pizza2.jpg	pizza
+teddy2.jpg	teddy
+teddy3.jpg	teddy
+teddy4.jpg	teddy
+toaster.jpg	toaster
+toaster2.png	toaster
+```
+
+The training and testing images are located in the assets folders. These images belong to Wikimedia Commons.
+> *[Wikimedia Commons](https://commons.wikimedia.org/w/index.php?title=Main_Page&oldid=313158208), the free media repository.* Retrieved 10:48, October 17, 2018 from:  
+> https://commons.wikimedia.org/wiki/Pizza  
+> https://commons.wikimedia.org/wiki/Toaster  
+> https://commons.wikimedia.org/wiki/Teddy_bear  
+
+## ML Task - [Image Classification](https://en.wikipedia.org/wiki/Outline_of_object_recognition)
+
+To solve this problem, first you'll build an ML model. Then, train the model on existing data, evaluate the model quality, and consume the model to classify a new image.
+
+## Create a console application
+
+### Create a project
+
+1. Open Visual Studio 2017. Select **File** > **New** > **Project** from the menu bar. In the **New Project** dialog, select the **Visual C#** node followed by the **.NET Core** node. Then select the **Console App (.NET Core)** project template. In the **Name** text box, type "TransferLearningTF" and then select the **OK** button.
+
+2. Create a directory named *Data* in your project to save your data set files:
+
+    In **Solution Explorer**, right-click on your project and select **Add** > **New Folder**. Type "Data" and hit Enter.
+
+3. Create a directory named *Models* in your project to save your model:
+
+    In **Solution Explorer**, right-click on your project and select **Add** > **New Folder**. Type "Models" and hit Enter.
+
+4. Install the **Microsoft.ML NuGet Package**:
+
+    In Solution Explorer, right-click on your project and select **Manage NuGet Packages**. Choose "nuget.org" as the Package source, select the Browse tab, search for **Microsoft.ML**. Click on the **Version** drop down, select the **0.10.0** package in the list, and select the **Install** button. Select the **OK** button on the **Preview Changes** dialog and then select the **I Accept** button on the **License Acceptance** dialog if you agree with the license terms for the packages listed. Repeat these steps for **Microsoft.ML.ImageAnalytics v0.10.0** and **Microsoft.ML.TensorFlow v0.10.0**.
+
+  > [!NOTE]
+  > This tutorial uses **Microsoft.ML v0.10.0**, **Microsoft.ML.ImageAnalytics v0.10.0**, and **Microsoft.ML.TensorFlow v0.10.0**.
+
+### Create classes and define paths
+
```

Comment:
Provide link to the sample code upfront, its very hard to follow these singular steps of copy pasting code. It's also error prone.